wcfsetup/install/files/lib/system/package/plugin/AbstractFileDeletePackageInstallationPlugin.class.php

```diff
@@ -98,6 +99,7 @@ protected function handleDelete(array $items)
             }
         }
 
+        $isCaseSensitiv = null;
```

Comment:
Should be `$isCaseSensitive`, shouldn't it? :)

Reply:
Thanks!